### PR TITLE
[JENKINS-42627] CLI is not translated to French

### PR DESCRIPTION
**Pull request by TristanFAURE**
https://issues.jenkins-ci.org/browse/JENKINS-42627
a suggestion to have jenkins CLI in french, 
i think the translation is relevant but it can obviously be discussed with other french native people (maybe french developer would also prefer to keep english messages ?)

### CODE REVIEW

File: cli/src/main/resources/hudson/cli/client/Messages_fr.properties [hunk 1]
```diff
@@ -0,0 +1,14 @@
+CLI.Usage=Jenkins CLI\n\
+  Utilisation: java -jar jenkins-cli.jar [-s URL] command [opts...] args...\n\
+  Options:\n\
+   -s URL       : l URL du serveur (par d�faut : variable d environnement JENKINS_URL)\n\
+   -i KEY       : fichier de la cl� priv�e SSH � utiliser pour l authentification\n\
```
Comment (batmat) [inline cli/src/main/resources/hudson/cli/client/Messages_fr.properties]: missing _apostrophe_

File: cli/src/main/resources/hudson/cli/client/Messages_fr.properties [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+   -p HOST:PORT : h�te et port des proxys HTTP et HTTPS. Voir https://jenkins.io/redirect/cli-https-proxy-tunnel\n\
+   -noCertificateCheck : contourne enti�rement la v�rification des certificats HTTPS. A utiliser avec pr�caution\n\
+   -noKeyAuth   : ne charge pas la cl� priv�e d authentification SSH. En conflit avec -i\n\
```
Comment (batmat) [inline cli/src/main/resources/hudson/cli/client/Messages_fr.properties]: missing _apostrophe_

File: cli/src/main/resources/hudson/cli/client/Messages_fr.properties [hunk 3]
```diff
@@ -0,0 +1,14 @@
...
+  \n\
+  Les commandes disponibles d�pendent du serveur. Lancer la commande 'help' pour\n\
+  obtenir la liste.
+CLI.NoURL=Ni -s ou la variable d environnement JENKINS_URL ne sont renseign�es.
+CLI.VersionMismatch=Conflit de versions. Cet outil ne peut fonctionner avec le serveur Jenkins renseign�.
+CLI.NoSuchFileExists=Ce fichier n existe pas: {0}
```
Comment (batmat) [inline cli/src/main/resources/hudson/cli/client/Messages_fr.properties]: apostrophe

**Review by batmat — CHANGES_REQUESTED** [review]
apostrophes missing

**Comment by TristanFAURE** [comment]
I first inserted apostrophe but at execution they were dropped. For instance the result was : 
LURL instead of L'URL that's why I dropped them.

**Comment by batmat** [comment]
@TristanFAURE as it is `.properties` you have to double the single quotes (apostrophe) to escape them.

**Comment by TristanFAURE** [comment]
Ok I will change that

File: cli/src/main/resources/hudson/cli/client/Messages_fr.properties [hunk 4]
```diff
@@ -0,0 +1,14 @@
...
+   -i KEY       : fichier de la cl\u00e9 priv\u00e9e SSH \u00e0 utiliser pour l''authentification\n\
+   -p HOST:PORT : h\u00f4te et port des proxys HTTP et HTTPS. Voir https://jenkins.io/redirect/cli-https-proxy-tunnel\n\
+   -noCertificateCheck : contourne enti\u00e9rement la v\u00e9rification des certificats HTTPS. A utiliser avec pr\u00e9caution\n\
+   -noKeyAuth   : ne charge pas la cl\u00e9 priv\u00e9e d''authentification SSH. En conflit avec -i\n\
+  \n\
+  Les commandes disponibles d\u00e9pendent du serveur. Lancer la commande 'help' pour\n\
+  obtenir la liste.
+CLI.NoURL=Erreur, ni -s ou la variable d''environnement JENKINS_URL ne sont renseign\u00e9es.
+CLI.VersionMismatch=Conflit de versions. Cet outil ne peut fonctionner avec le serveur Jenkins renseign\u00e9.
+CLI.NoSuchFileExists=Ce fichier n''existe pas: {0}
```
Comment (batmat) [inline cli/src/main/resources/hudson/cli/client/Messages_fr.properties]: NIT: en français, ya une espace avant deux points (:), mais je ne vais pas/plus bloquer juste pour ça.

Thanks!

**Pull request merged**